src/reanimated2/ViewDescriptorsSet.ts

```diff
@@ -23,6 +24,8 @@ export interface ViewDescriptorsSet {
   ) => void;
 }
 
+const scheduleUpdates: Platform.OS === 'web' ? requestAnimationFrame : setImmediate;
```

Comment:
Shouldn't it be?

```ts
const scheduleUpdates = Platform.OS === 'web' ? requestAnimationFrame : setImmediate;
```

I think it's the reason of failing `eslint`

Reply:
I updated it to match this line: https://github.com/DrRefactor/react-native-reanimated/blob/54fb4137ac6697e322d54ece10b1a7a0260d6bfa/src/core/AnimatedNode.js#L79